apps/scan/frontend/src/screens/scan_error_screen.tsx

```diff
@@ -78,19 +146,21 @@ export function ScanErrorScreen({
       voterFacing
     >
       <FullScreenPromptLayout
-        title={appStrings.titleScannerBallotNotCounted()}
+        title={title}
         image={
           <FullScreenIconWrapper>
             <Icons.Delete color="danger" />
           </FullScreenIconWrapper>
         }
       >
         <P>{errorMessage}</P>
-        {restartRequired ? (
-          <P>{appStrings.instructionsScannerAskForRestart()}</P>
-        ) : (
-          <Caption>{appStrings.noteAskPollWorkerForHelp()}</Caption>
-        )}
+        <Caption>
+          {pollWorkerMessage === 'needs-help'
+            ? appStrings.instructionsAskForHelp()
+            : pollWorkerMessage === 'might-need-help'
+            ? appStrings.noteAskPollWorkerForHelp()
```

Comment:
Some issues require poll worker intervention. Others don't, but a voter could still ask for help. I wanted to reflect this in the copy. It seems silly for a voter to "ask a poll worker _if_ they need help" if the problem is something like a mismatched ballot hash they have no control over.